Assets/Creatures/impl/Playable/Zephyr/Abilities/charged/Vortex.cs

```diff
@@ -108,13 +123,20 @@ private void UpdateVortexPosition()
             Vector3 mousePosition = Input.mousePosition;
             mousePosition.z = 10.0f;
             Vector3 worldPosition = mainCamera.ScreenToWorldPoint(mousePosition);
+            Vector3 adjustedPosition = mainCamera.ScreenToWorldPoint(mousePosition);
+            adjustedPosition.y += positionYAdjustment;
+            adjustedPosition.x += positionXAdjustment;
+
+            // Make the tornado lag behind the cursor a bit
+            Vector3 smoothPosition = Vector3.MoveTowards(vortex.transform.position, adjustedPosition, followSpeed * Time.deltaTime);
+
             if (vortex != null)
             {
-                vortex.transform.position = worldPosition;
-                channelTornadoEffect.transform.position = worldPosition;
-                vortexEffect.transform.position = worldPosition;
-                smokeRingEffect.transform.position = worldPosition;
-                Debug.Log($"Vortex position updated to {worldPosition}");
+                vortex.transform.position = smoothPosition;
```

Comment:
That's a good point, tbh this code is probably going to need to change because it didn't work either 🤣 

I have three separate animations and was trying to get them to play simultaneously but it only ever shows one of them